env.default

```diff
@@ -52,3 +52,9 @@ BASKET_SQS_QUEUE_URL=
 RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
 RECAPTCHA_SECRET_KEY=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
 RECAPTCHA_ENABLED=False
+
+# SQS for Basket
+BASKET_QUEUE_REGION=
```

Comment:
We don't currently use this environment variable in the code base, on the basis that the URL should include the region? 

The queue URL is currently fetched from `BASKET_SQS_QUEUE_URL` (not `BASKET_QUEUE_URL`) - see line 48 above. We'll need to change one or the other. We also need to specify a root URL for the API (line 47 above).

We also don't currently use either of `BASKET_ACCESS_KEY` and `BASKET_SECRET_ACCESS_KEY` - can you confirm what these are for? From what I could tell the newsletter signup API doesn't require authentication?